### DISCUSSION TITLE
Moderate severity after Zigbee2mqtt update

### DISCUSSION THREAD

**Discussion by wildchai** [tteck/Proxmox#1783] [Help Needed] [answered]
Hi, 
After updating the Zigbee2mqtt container using the script. It shows these messages. What is the issue and should it be fixed and how to go about it? Thanks. I am new to proxmox and been using all tteck's script. 

**1 moderate severity vulnerability

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
Restoring configuration...
Starting Zigbee2MQTT...
Done!
root@zigbee2mqtt:~# npm audit
npm ERR! code ENOLOCK
npm ERR! audit This command requires an existing lockfile.
npm ERR! audit Try creating one first with: npm i --package-lock-only
npm ERR! audit Original error: loadVirtual requires existing shrinkwrap file

npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-09-10T06_52_55_562Z-debug-0.log**

**Comment by tteck** [comment, marked as answer]
```
npm cache clean --force
npm audit fix --force
```
